### PR TITLE
Make defaults object accessible from outside

### PR DESCRIPTION
**Pull request by samosad**

### CODE REVIEW

**Comment by jackocnr** [comment]
Would you be able to add a test for this?

**Comment by samosad** [comment]
done,
after I added a test, I found a problem - if defaults object is accessible from outside than its keys shouldn't be replaced with one-symbol names. So I removed some code from `replace` task to make tests pass again.

File: src/spec/tests/static/defaults.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+describe("defaults: change default plugin options", function() {
+
+	var defaults = $.extend({}, $.fn.intlTelInput.defaults);
```
Comment (jackocnr) [inline src/spec/tests/static/defaults.js]: Did you mean to use this somewhere? I guess in the afterEach, to reset the defaults again?

Comment (samosad) [inline src/spec/tests/static/defaults.js]: yes, my bad) fixed

**Comment by jackocnr** [comment]
Released in v9.2.0 - thanks again!

**Pull request merged**